PYME/Acquire/Hardware/AndorNeo/AndorZyla.py

```diff
@@ -130,26 +143,9 @@ def noise_properties(self):
                     }
 
 
-    # this class is compatible with the ATEnum object properties that are used in ZylaControlPanel
-    # we use it as a higher level alternative to setting gainmode and encoding directly
-    class SimpleGainEnum(object):
-        def __init__(self, cam):
-            self.cam = cam
-            self.gainmodes = cam.PixelEncodingForGain.keys()
-            self.propertyName = 'SimpleGainModes'
-            
-        def getAvailableValues(self):
-            return self.gainmodes
-
-        def setString(self,str):
-            self.cam.SetSimpleGainMode(str)
-
-        def getString(self):
-            return self.cam.GetSimpleGainMode()
-
-
```

Comment:
I recall there being a good reason for this (which might have evaporated with a more recent SDK3 version), but can't put my finger on it. It could be related to Andor Neo support.

Reply:
This is needed - and is not just an alias for `SimplePreAmpGainControl`, as `SetSimpleGainMode()` actually sets two properties (the gain mode **and** pixel encoding which is not set automatically by setting the simple PreAmpGainControl property).

Reply:
I reinstated SimplePreampGainControl and see the reason for the pixel encoding. Interestingly, with the latest SDK there were no issues not doing it although I can't be sure it was stuck on 16bit. Anyway, seems to work fine now with latest commits.